src/stylesheets/base/_config.scss

```diff
@@ -24,14 +24,14 @@ $colors: (
 
 $screens: (
   sm:                   'min-width: 31.25rem', // 500px
-  SM:                   'max-width: 31.25rem',
+  sm-max:               'max-width: 31.25rem',
```

Comment:
Two other small things to keep in mind for media queries in future work are:

1. Using the same exact value for both min and max width media queries could lead to competing rules in your css, for example:

```css
@media (screen(sm) { ... }
@media (screen(sm-max) { ... }
```

Outputs:
```css
@media ('min-width: 31.25rem') { ... }
@media ('max-width: 31.25rem') { ... }
```

Which of those media queries applies to a viewport size of 500px? Both! Since the value is the same. To get around this, you'd want to make the value be 1px different (see below). Let me know if this doesn't make sense.
```sass
$screens: (
  sm:     'min-width: 31.25rem',   // 500px
  sm-max: 'max-width: 31.1875rem', // 499px
  ...
)
```

Also:

2. When using relative units for media query breakpoints, we typically use `em` values, not `rem` values, although I'm not 100% sure if there is an actual difference between em and rem media queries.

Reply:
Makes total sense! Can't believe I overlooked this, but thanks for catching it. Curious about the `em` vs `rem` debate though. Is there any harm in using `rem` for breakpoints? I didn't research this terribly much, however, I came across [this post](https://adamwathan.me/dont-use-em-for-media-queries/) by the Tailwind guy who noticed an issue with `em` breakpoints in Safari.

Reply:
Unfortunately there's A LOT of conflicting info out there on px vs em based media queries, and not a lot about rem afaik.

Reply:
This article mentions `em` and `rem` media queries at the end and says that there's no difference between them in the tests he ran. https://zellwk.com/blog/media-query-units/